docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -17,27 +17,57 @@ Give the flag a descriptive human readable name and create a single variation fo
 
 ![Feature gate 1](/img/feature-flagging/feature-gate-1.png)
 
-### Create allocations to describe your target audience
+### 2. Create allocations to describe your target audience
 
 After creating the flag, decide who to target by creating [allocations](/feature-flags#allocations). In this case we will create two allocations that describe our target audience for the new page: internal users and half of all North American web users:
 
 ![Feature gate 2](/img/feature-flagging/feature-gate-2.png)
 
 Create each allocation one by one, giving each a name and specifying the traffic split and traffic exposure. Finally, enter rules that specify which subjects are part of that allocation. In the example above `Internals users` are determined by their email, and `North America Web Users` are determined by their country and device. For this allocation we'll set the traffic exposure to 50% so that 50% of the group still sees the original page (SDK will return `null`).
```

Comment:
Curious for @vpai 's take on this. I think we should not put a 50/50 split with a NULL group in the docs. When you are doing a 50/50 split you are going to run an experiment, and there should be explicit variants (if only for clarity). Also would we even be logging the NULL assignment in this case?

Reply:
Could also argue that it is overkill to create two allocations before even setting up the SDK in a quick start. The whole point is to get end-to-end as quickly as possible. Could talk about the second allocation as an optional 4th part

Reply:
Yep agreed on the first one (actually wrote about that if the FF common issues doc).

I think showing that you can create multiple allocations is nice, if a user wants to skip the second that's fine